controllers/history.controller.js

```diff
@@ -0,0 +1,88 @@
+const History = require("../models/history-model");
+const User = require("../models/user-model");
+const { concat,remove }=require("lodash");
+
+const getHistory=async(req,res)=>{
+  const history=await History.find({});
+  res.json({success:true,history});
```

Comment:
I think you forgot to add the status code here

Reply:
done